js/big-photo.js

```diff
@@ -34,13 +30,7 @@ function buttonClosePhotoKeydownHandler (evt) {
 function makeTextInfo (photo) {
```

Comment:
```suggestion
function makeBigPictureInfo (photo) {
```

ведь мы делаем информацию для большой фотки, а не для текста